src/Packetery/Module/Options/FlagManager/FeatureFlagProvider.php

```diff
@@ -60,12 +60,8 @@ public function __construct(
 	 * @throws Exception From DateTimeImmutable.
 	 */
 	public function isSplitActive(): bool {
-		$flags = $this->featureFlagDownloader->getFlags();
-		if ( isset( $flags[ self::FLAG_SPLIT_ACTIVE ] ) ) {
-			return (bool) $flags[ self::FLAG_SPLIT_ACTIVE ];
-		}
-
-		return false;
+		// Temporarily enabled. Split will be removed completely in the 2nd phase.
```

Comment:
Napsal bych spíš třeba "Enabled for all users. Method will be removed later."

Reply:
Upraveno.